lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -586,7 +586,7 @@ def build_mark_for_capture_xml(money, authorization, parameters = {})
         xml.tag! :Request do
           xml.tag! :MarkForCapture do
             add_xml_credentials(xml)
-            xml.tag! :OrderID, format_order_id(order_id)
+            xml.tag! :OrderID, format_order_id(order_id || parameters[:order_id])
```

Comment:
In the linked issue you mention

> "capture" test cases were rejected because I did not provide the OrderID.

Why is it that the `order_id` from the authorization did not work in your case? Line 584 (the top of this method) attempts to extract it from the authorization, which is what would be typically expected in an auth and capture flow. Is that behavior buggy? It seems weird to me that you'd have to pass this in separately when trying to capture an auth.